clang-tools-extra/clangd/Hover.cpp

```diff
@@ -1488,6 +1491,8 @@ markup::Document HoverInfo::present() const {
           llvm::formatv(" (+{0} padding)", formatSize(*Padding)).str());
     }
   }
+  if (Align)
+    Output.addParagraph().appendText("Align: " + formatSize(*Align));
```

Comment:
My preference would be to have alignment on the same line as size, so that less vertical space is wasted. The following hover should illustrate why this might be more important than it appears on other hovers posted in this PR: 
![image](https://github.com/llvm/llvm-project/assets/12883766/426a6770-3675-46eb-9f33-0d01429f7aa8)

Reply:
> My preference would be to have alignment on the same line as size

But size may include padding for member fields. How would you like to see it?

Like this?
```
Size 24 bytes (8 bytes align) // for structs/classes
Size 1 byte (+7 bytes padding, 8 bytes align) // for member fields with padding
```

Reply:
That would work, but I'd prefer `Size 24 bytes, alignment 8 bytes` wording. I don't think alignment as a property of type is secondary to size to put it in parentheses.

Reply:
Yeah, it's looks better

![image](https://github.com/llvm/llvm-project/assets/12231048/a58f9b99-f8d4-4cf3-8b24-fb251136791b)
![image](https://github.com/llvm/llvm-project/assets/12231048/ac9d1693-fc65-453a-9160-be466f99d481)